### DISCUSSION TITLE
Add attrs in rio.to_raster

### DISCUSSION THREAD

**Discussion by Graziottomp** [corteva/rioxarray#665] [Q&A]
Hi. I', converting some netcdf files to geotiff. I manege to do it, but I also want to add some attributs to the raster file. It is possible to do whitout open the raster file after the convertion line?

**Comment by snowman2** [comment]
If you want to add attributes to the raster file, you can do it beforehand by adding `.attrs` to your dataset/dataarray.

If you want to modify your existing file, you can do it directly with rasterio using the `r+` mode.
https://rasterio.readthedocs.io/en/latest/topics/tags.html#writing-tags
